app/gui/src/dashboard/providers/BackendProvider.tsx

```diff
@@ -110,15 +110,28 @@ export function useBackend(category: Category) {
   const remoteBackend = useRemoteBackend()
```

Comment:
i know it hasn't been edited in this PR but the error message above looks wrong

Reply:
Fixed, thanks!